.github/workflows/sveltekit.yaml

```diff
@@ -6,8 +6,11 @@ on:
   push:
     branches:
       - master
+  pull_request_review:
```

Comment:
I think we may want this to run whenever a pull request is submitted, too. As a reviewer, I'd like to know the code doesn't crash the build _before_ I approve a PR. 

What if we triggered on `pull_request` rather than `pull_request_review`. In that case, you may not need the conditional on line 13.